README.md

```diff
@@ -534,6 +534,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Kai Bechdel](https://github.com/therealkai)
   - [Kaiwalya Koparkar](https://github.com/kaiwalyakoparkar)
   - [Kanishk Pachauri](https://github.com/Mr-Sunglasses)
+  - [Kanhaiya Bhatt](https://github.com/kb-s)
```

Comment:
Please add your name in line 536, after Kaiwalya Koparkar.

Reply:
@Panquesito7 thanks.